packages/backend-common/src/dynamodb.ts

```diff
@@ -51,3 +55,23 @@ export const writeTranscriptionItem = async (
 		throw error;
 	}
 };
+
+export const getTranscriptionItem = async (
+	client: DynamoDBDocumentClient,
+	tableName: string,
+	itemId: string,
+) => {
+	const command = new GetCommand({
+		TableName: tableName,
+		Key: {
+			id: itemId,
+		},
+	});
+	try {
+		const result = await client.send(command);
+		return result.Item;
+	} catch (error) {
+		console.error(`Failed to get item ${itemId} from dynamodb`, error);
+		return null;
```

Comment:
I think in general TypeScript guidelines recommends using `undefined` rather than null wherever possible. Do you think we can change this to be undefined?